app/controllers/IndexController.scala

```diff
@@ -17,23 +17,36 @@
 package controllers
 
 import controllers.actions.IdentifierAction
+import models.UserAnswers
+import play.api.Logging
 
 import javax.inject.Inject
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
 import views.html.IndexView
 
+import scala.concurrent.ExecutionContext
+
 class IndexController @Inject() (
   val controllerComponents: MessagesControllerComponents,
+  sessionRepository: SessionRepository,
   identify: IdentifierAction,
   view: IndexView
-) extends FrontendBaseController
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with Logging
     with I18nSupport {
 
-  def onPageLoad: Action[AnyContent] = identify {
+  def onPageLoad: Action[AnyContent] = identify.async {
     implicit request =>
-      Ok(view())
+      sessionRepository.set(UserAnswers(request.userId)) map {
+        case true => Ok(view())
+        case false =>
+          logger.error(s"Failed to initialize user answers for userId: [${request.userId}]")
```

Comment:
is this so we dont use the index as a landing page before starting the journey?

Reply:
It's so the userAnswers object is initialised when the index page is visited. This is to ease testing by the testers until we have the full journey implemented.